sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/DefaultSchedulerTest.java

```diff
@@ -61,6 +63,7 @@
  * This class tests the DefaultScheduler class.
  */
 @SuppressWarnings({"PMD.TooManyStaticImports", "PMD.AvoidUsingHardCodedIP"})
+@Ignore
```

Comment:
Yes, there's some flakiness with mocks that @ArtRand is going to take a look at when he gets the chance, then it'll be reenabled.